e2e/nomostest/ntopts/gke.go

```diff
@@ -56,25 +56,25 @@ func GKECluster(t testing.NTB, apiServerTimeout time.Duration) Opt {
 func forceAuthRefresh(t testing.NTB) {
 	out, err := exec.Command("kubectl", "config", "current-context").CombinedOutput()
 	if err != nil {
-		t.Fatalf("failed to get current config: %v", err)
+		t.Fatalf("failed to get current config (stdout: %s): %v", string(out), err)
```

Comment:
`CombinedOutput` returns both stdout and stderr, so might be clearer to mention both. I also expect the stdout/stderr could be verbose, so I'd suggest putting it at the end of the string. e.g.
```
t.Fatalf("failed to get current config: %v\nstdout/stderr:\n%s", err, string(out))
```

Reply:
done